src/PerfView/UserCommands.cs

```diff
@@ -1512,6 +1533,37 @@ private TraceEvents GetTraceEventsWithProcessFilter(ETLDataFile etlFile)
             return events;
         }
 
+        /// <summary>
+        /// Save the CPU stacks for an ETL file into a perfView.xml.zip file.
+        /// </summary>
+        /// <param name="etlFile">The ETL file to save.</param>
+        /// <param name="process">The process to save. If null, save all processes.</param>
+        /// <param name="filter">The filter to apply to the stacks. If null, apply no filter.</param>
+        /// <param name="outputName">The name of the file to output data to. If null, use the default.</param>
+        private static void SaveCPUStacksForProcessAsCsv(ETLDataFile etlFile, TraceProcess process = null, FilterParams filter = null, string outputName = null)
+        {
+            // Focus on a particular process if the user asked for it via command line args.
+            if (process != null)
+            {
+                etlFile.SetFilterProcess(process);
+            }
+
+            if (filter == null)
+            {
+                filter = new FilterParams();
+            }
+
+            var stacks = etlFile.CPUStacks();
+            stacks.Filter = filter;
+            stacks.LookupWarmSymbols(10); // Look up symbols (even on the symbol server) for modules with more than 50 inclusive samples
```

Comment:
nit pick but you can replace "with more than 50 inclusive samples" in the comment with "with at least 10 inclusive samples"

Reply:
Fixed